oeps/processes/oep-0055/decisions/0003-readme-specification.rst

```diff
@@ -0,0 +1,183 @@
+0003 README specification
+#########################
+
+Status
+******
+
+** Accepted **
+
+Context
+*******
+
+`OEP-55`_ states that one of the jobs of a maintainer is te maintain an
+up-to-date and useful readme. While it laid out some possibilites about what
+should go into a README, it did not go into much detail.
+
+Decision
+********
+
+We will define a specification that READMEs in the open edx project must
+follow.  By having a single definition, we can make it easier for people to
+navigate between projects and find the most important information quickly.
+
+Specification
+*************
+
+The readme of a well-maintained repo should contain at least the following
+sections.  Ideally in this exact order.
```

Comment:
Assuming we didn't think of all sections and usages for future (or event current READMEs), may there should be a process noted here for:
1. Where to insert new custom headers when you feel they are warranted.
2. Whether there should be a process for capturing popular optional headers to help provide consistency.

Here is an example of where I just updated a README and wanted to provide a little warning around installation, and ad-hoc made up and inserted a new header: https://github.com/openedx/event-bus-kafka/blame/main/README.rst#L89. Would you see this as a subsection on one of your proposed headers? Other?

Also, if there are standard or recommended subsections, etc. they could be detailed here. Related, these intro sentences could mention that there is a template below that may provide more clarity for each header.